x-pack/plugins/security_solution/cypress/screens/alerts_details.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const JSON_CONTENT = '.ace_content';
+
+export const JSON_VIEW_TAB = '#json-view';
```

Comment:
We should be able to add `data-test-subj`s for these tabs, but it looks like it would have to go into the tab's contents.